src/map.rs

```diff
@@ -165,10 +165,12 @@ pub async fn start_uds_server<T>(m: T) -> Result<(), Box<dyn std::error::Error>>
 where
     T: Mapper + Send + Sync + 'static,
 {
-    shared::write_info_file();
+    shared::write_info_file().map_err(|e| format!("writing info file: {e:?}"))?;
 
     let path = "/var/run/numaflow/map.sock";
-    std::fs::create_dir_all(std::path::Path::new(path).parent().unwrap())?;
+    let path = std::path::Path::new(path);
+    std::fs::create_dir_all(path.parent().unwrap())
+        .map_err(|e| format!("creating directory /var/run/numaflow : {e:?}"))?;
```

Comment:
```suggestion
    let path = std::path::Path::new(path);
    let parent = path.parent().unwrap();
    std::fs::create_dir_all(parent)
        .map_err(|e| format!("creating directory {parent} : {e:?}"))?;
```

let's not hardcode in 2 places